CHANGELOG.md

```diff
@@ -7,16 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Added
+### Changed
 
-- New `initial_system_message` field in `FlowConfig`, which allows setting a
-  global system message for static flows.
+- Nodes now have two message types to better delineate defining the role or
+  persona of the bot from the task it needs to accomplish. The message types are:
 
-### Changed
+  - `role_message`, which defines the personality or role of the bot
+  - `task_message`, which defines the task to be completed for a given node
```

Comment:
Should these say `role_messages` and `task_messages`? Or simply "Role message, which..." and "Task message, which...". Just mentioning since `role_message` is not an actual thing.

Reply:
Should be `...messages`. Thanks for catching that.